examples/missing-data/os-01-dr-02-dc-05.json

```diff
@@ -0,0 +1,43 @@
+[
+  {
+    "statementID": "e9c5bedd-2627-4f16-ac0c-a1b7ebfa051d",
+    "statementType": "entityStatement",
+    "statementDate": "2018-12-17",
+    "entityType": "registeredEntity",
+    "name": "Listed Company OS-17",
+    "identifiers": [
+      {
+        "scheme": "GB-COH",
+        "id": "XE000017"
+      }
+    ]
+  },
+  {
+    "statementID": "4f683208-f571-40a5-8a5a-ec78c2930387",
+    "statementType": "personStatement",
+    "statementDate": "2018-12-17",
+    "personType": "anonymousPerson",
+    "unspecifiedPersonDetails": {
+      "reason": "interested-party-exempt-from-disclosure",
+      "description": "super-secure-person-with-significant-control"
+    }
+  },
+  {
+    "statementID": "a63852a4-ee66-44b2-870c-747692c4d393",
+    "statementType": "ownershipOrControlStatement",
+    "statementDate": "2018-12-17",
+    "subject": {
+      "describedByEntityStatement": "e9c5bedd-2627-4f16-ac0c-a1b7ebfa051d"
+    },
+    "interestedParty": {
+      "describedByPersonStatement": "4f683208-f571-40a5-8a5a-ec78c2930387"
+    },
+    "source": {
+      "assertedBy": [
+        {
+          "name": "Company A"
+        }
+      ]
+    }
+  }
+]
```

Comment:
For these two new example files, we'll need to add the new publicationDetails block to them ultimately. You could add them on this branch and merge PR #193 first?